src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/TaxRate.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="%sylius.model.tax_rate.class%" shortName="TaxRate">
+        <attribute name="route_prefix">admin</attribute>
+
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>tax_rate:read</attribute>
```

Comment:
for all the rest of our resources, we have `admin` or `shop` prefixes. I would prefer to stick to it so that we will have coherence. Perhaps we should write an ADR about it. The point is, that, for me rule that enforces the existence of `admin` and `shop` as serialization prefixes is easier to follow than the rule, that we sometimes don't write them. 
 
```suggestion
                <attribute>admin:tax_rate:read</attribute>
```

But going further from it, I'm against defining admin serialization groups as a default one. I would have to check, but if I recall correctly, this group will also be assigned to the configuration if the config is overridden. Therefore, if that is true, we will assign serialization groups that may leak lead to some data leaks. Therefore, all together, I would recommend putting `normalization_context` operation by operation

Reply:
You're right. I missed that during refactoring the previous PR. It'll be pushed soon.